### ISSUE TITLE
Error setting up database

### ISSUE THREAD

**Issue by TigerWolf** [envato/event_sourcery-postgres#60]
Im using the todo app example (https://github.com/envato/event_sourcery_todo_app) and I am having a problem setting up my database. The error is:

```bundle exec rake db:migrate
rake aborted!
Sequel::DatabaseError: PG::InvalidParameterValue: ERROR:  identity column type must be smallint, integer, or bigint
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:152:in `async_exec'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:152:in `block in execute_query'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/database/logging.rb:38:in `log_connection_yield'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:152:in `execute_query'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:140:in `block in execute'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:116:in `check_disconnect_errors'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:140:in `execute'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:496:in `_execute'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:314:in `block (2 levels) in execute'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:518:in `check_database_errors'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:314:in `block in execute'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/connection_pool/threaded.rb:91:in `hold'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/database/connecting.rb:270:in `synchronize'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:314:in `execute'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/database/query.rb:50:in `execute_dui'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/database/query.rb:43:in `execute_ddl'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/database/schema_methods.rb:682:in `create_table_from_generator'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/database/schema_methods.rb:194:in `create_table'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/event_sourcery-postgres-0.8.0/lib/event_sourcery/postgres/schema.rb:55:in `create_aggregates'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/event_sourcery-postgres-0.8.0/lib/event_sourcery/postgres/schema.rb:17:in `create_event_store'
/home/ubuntu/environment/event_sourcery_todo_app/Rakefile:85:in `block (2 levels) in <top (required)>'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
/home/ubuntu/.rvm/gems/ruby-2.6.3/bin/ruby_executable_hooks:24:in `eval'
/home/ubuntu/.rvm/gems/ruby-2.6.3/bin/ruby_executable_hooks:24:in `<main>'

Caused by:
PG::InvalidParameterValue: ERROR:  identity column type must be smallint, integer, or bigint
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:152:in `async_exec'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:152:in `block in execute_query'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/database/logging.rb:38:in `log_connection_yield'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:152:in `execute_query'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:140:in `block in execute'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:116:in `check_disconnect_errors'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:140:in `execute'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:496:in `_execute'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:314:in `block (2 levels) in execute'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:518:in `check_database_errors'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:314:in `block in execute'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/connection_pool/threaded.rb:91:in `hold'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/database/connecting.rb:270:in `synchronize'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/adapters/postgres.rb:314:in `execute'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/database/query.rb:50:in `execute_dui'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/database/query.rb:43:in `execute_ddl'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/database/schema_methods.rb:682:in `create_table_from_generator'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/sequel-5.14.0/lib/sequel/database/schema_methods.rb:194:in `create_table'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/event_sourcery-postgres-0.8.0/lib/event_sourcery/postgres/schema.rb:55:in `create_aggregates'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/event_sourcery-postgres-0.8.0/lib/event_sourcery/postgres/schema.rb:17:in `create_event_store'
/home/ubuntu/environment/event_sourcery_todo_app/Rakefile:85:in `block (2 levels) in <top (required)>'
/home/ubuntu/.rvm/gems/ruby-2.6.3/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
/home/ubuntu/.rvm/gems/ruby-2.6.3/bin/ruby_executable_hooks:24:in `eval'
/home/ubuntu/.rvm/gems/ruby-2.6.3/bin/ruby_executable_hooks:24:in `<main>'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)```

My environment is:
ruby: 2.6.3
postgres: 10.9

The app still runs but it appears that the `writeEvents` function is not created which means I cannot add any events.

**Comment by TigerWolf** [comment]
I think this might be a duplicate of #59 but I am using Postgres 10 rather than 11. Looking into gem versions as well to see if I can fix it.

**Comment by zubin** [comment]
Resolved by #59.
